src/GitUtilities.js

```diff
@@ -129,9 +130,12 @@ export default class GitUtilities {
   }
 
   static diffSinceIn(since, location, opts) {
-    log.silly("diffSinceIn", since, location);
+    const formattedLocation = path.relative(opts.cwd, location).replace(/\\/g, '/');
+    const diff = ChildProcessUtilities.execSync("git", [
+      "diff", "--name-only", since, "--", formattedLocation
+    ], opts);
 
-    const diff = ChildProcessUtilities.execSync("git", ["diff", "--name-only", since, "--", location], opts);
+    log.silly("diffSinceIn", since, formattedLocation);
```

Comment:
Please move this first `log.silly` above the execSync call (but obviously after the `formattedLocation` initialization).